### PR TITLE
Fcnn paper

### PR DESCRIPTION
**Pull request by jodoin**
Fcnn paper + new bounding-box tag + localization tag for YOLO

### CODE REVIEW

File: deep-learning/_posts/2017-03-30-fcnn.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+tags: deep-learning CNN essentials segmentation
+cite:
+    authors: "J. Long, E. Shelhamer, T. Darrell"
+    title:   "Fully Convolutional Networks for Semantic Segmentation"
+    venue:   "proceedings of CVPR 2015, p.3431-3440"
+pdf:   "http://www.cv-foundation.org/openaccess/content_cvpr_2015/html/Long_Fully_Convolutional_Networks_2015_CVPR_paper.html"
+---
+
+ 
+One of the first and most successful CNN-based segmentation paper.  The proposed model is a simple concatenation of convolution and maxpooling layers but with one (or more) up-sampling (or deconvolution) layers at the end.  The reasons for these up-sampling layers is to force the output to have the same size than the input and thus to have a segmentation map with the right size.
```
Comment (lemairecarl) [inline deep-learning/_posts/2017-03-30-fcnn.md]: "concatenation of convolution and maxpooling layers" --> I suggest replacing "concatenation" with "sequence"

File: deep-learning/_posts/2017-03-30-fcnn.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+![](/deep-learning/images/fcnn/fcnn_1.jpg)
+
+They also propose to use skip connexions (similar to those in the [U-Net](https://vitalab.github.io/deep-learning/2017/02/27/unet.html)) which allows to have a more progression transition from coarse features to the end segmentation map.
```
Comment (lemairecarl) [inline deep-learning/_posts/2017-03-30-fcnn.md]: connexions --> connections
more progression transition --> more progressive transition

File: deep-learning/_posts/2017-03-30-fcnn.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+tags: deep-learning CNN essentials segmentation
+cite:
+    authors: "J. Long, E. Shelhamer, T. Darrell"
+    title:   "Fully Convolutional Networks for Semantic Segmentation"
+    venue:   "proceedings of CVPR 2015, p.3431-3440"
+pdf:   "http://www.cv-foundation.org/openaccess/content_cvpr_2015/html/Long_Fully_Convolutional_Networks_2015_CVPR_paper.html"
+---
+
+ 
+One of the first and most successful CNN-based segmentation paper.  The proposed model is a simple sequence of convolution and maxpooling layers but with one (or more) up-sampling (or deconvolution) layers at the end.  The reasons for these up-sampling layers is to force the output to have the same size than the input and thus to have a segmentation map with the right size.
```
Comment (czotti) [inline deep-learning/_posts/2017-03-30-fcnn.md]: I'll remove deconvolution, because `upsampling` and `deconvolution` is two different things.

Comment (jodoin) [inline deep-learning/_posts/2017-03-30-fcnn.md]: done

**Pull request merged**